test/numerical/TestLSTM.cpp

```diff
@@ -87,17 +113,25 @@ int main(int argc, char *argv[]) {
             for (int64_t dynb = 0; dynb < 2; dynb++)
               for (int64_t noneh = 0; noneh < 2; noneh++)
                 for (int64_t nonec = 0; nonec < 2; nonec++)
+                // clang-format off
+#ifdef TEST_NONEP_ONLY
+                  {
+                    int64_t nonep = 1;
+#else
                   for (int64_t nonep = 0; nonep < 2; nonep++) {
+#endif
```

Comment:
use the loop in all case, but define outside of the loops the min value to be 0 or 1

Reply:
Done. Thanks!